packages/localize-tools/src/modes/runtime.ts

```diff
@@ -58,6 +58,21 @@ async function runtimeOutput(
   config: Config,
   runtimeConfig: RuntimeOutputConfig
 ) {
+  let fileExtension;
+  const language = runtimeConfig.language ?? 'ts';
```

Comment:
Is this the real default? Seems like we should only default to TS when there's a tsconfig file, but that may be happening elsewhere.

Reply:
Good point! I was thinking we'd need to default to `ts` to avoid a breaking change, but since `tsConfig` was required before, we can make `js` the default when it is not set, without a breaking change.

Done.